crates/extensions/c8y_config_manager/src/actor.rs

```diff
@@ -433,10 +433,6 @@ impl ReceiveMessages<ConfigInput> for ConfigManagerMessageBox {
 impl MessageBox for ConfigManagerMessageBox {
```

Comment:
ConfigManagerMessageBox doesnt log currently log so I've left it as DynSender rather than logging sender but I could change it. but maybe the lack of logging was a mistake.

Reply:
> ConfigManagerMessageBox doesnt log currently log so I've left it as DynSender rather than logging sender but I could change it. but maybe the lack of logging was a mistake.

I think it was a mistake.